examples/test.yaml

```diff
@@ -0,0 +1,23 @@
+type: prompt-engine
+config:
+  model_config:
+    max_tokens: 1024
+  description_prefix: "Hello"
+  description_postfix: "!"
+  newline_operator: "\n"
+  input_prefix: ">"
```

Comment:
For chat, I think it's better to make this a dialog between two named agents. E.g.: 

```
input_prefix: "Human:"
input_postfix: ""
output_prefix: "Bot:"
output_postfix: ""
```

Reply:
Pushed in latest commit